apps/wallet-mobile/src/HW/LedgerConnect/LedgerConnect.tsx

```diff
@@ -160,6 +168,7 @@ class _LedgerConnect extends React.Component<Props, State> {
       })
       await onConnectBLE(device.id.toString())
     } catch (e) {
+      if (!(e instanceof Error)) return
```

Comment:
Yap, it's more about what ledger is throwing if it's throwing, `class MyError` and is not extending `class MyError extends Error`, it can break something, or even `const MyError = () => whatever not Error`